flink-python/pyflink/common/typeinfo.py

```diff
@@ -720,6 +720,39 @@ def __repr__(self) -> str:
         return 'MapTypeInfo<{}, {}>'.format(self._key_type_info, self._value_type_info)
 
 
+class LocalTimeTypeInfo(TypeInformation):
+
+    class TimeType(Enum):
+        LOCAL_DATE = 0
+        LOCAL_TIME = 1
+        LOCAL_DATE_TIME = 2
+
+    def __init__(self, time_type: TimeType):
+        super(LocalTimeTypeInfo, self).__init__()
+        self._time_type = time_type
+        self._j_typeinfo = None
```

Comment:
We have set `self._j_typeinfo = None` in the base class `TypeInformation`, so we can remove this line